src/sinks/new_relic_logs.rs

```diff
@@ -79,23 +79,7 @@ inventory::submit! {
 
 impl GenerateConfig for NewRelicLogsConfig {
     fn generate_config() -> toml::Value {
-        toml::Value::try_from(Self::with_encoding(Encoding::Json)).unwrap()
-    }
-}
-
-#[derive(Deserialize, Serialize, Debug, Eq, PartialEq, Clone, Derivative)]
-#[serde(rename_all = "snake_case")]
-#[derivative(Default)]
-pub enum Encoding {
-    #[derivative(Default)]
-    Json,
-}
-
-impl From<Encoding> for crate::sinks::http::Encoding {
-    fn from(v: Encoding) -> crate::sinks::http::Encoding {
-        match v {
-            Encoding::Json => crate::sinks::http::Encoding::Json,
-        }
+        toml::Value::try_from(Self::default()).unwrap()
```

Comment:
Why is the `unwrap` safe here? I might make it `expect(...)`.

Reply:
Aha 👍 Fair enough. It just wasn't immediately clear why this was safe. I still think we should generally prefer `expect()` over `unwrap()` so we can document why it is safe.